src/main/resources/Task 1 - Shoe Object Modelling.txt

```diff
@@ -0,0 +1,5 @@
+Object Modeling Shoes:
+- create a class diagram for shoes object hierarchy.
+- implement class diagram in Java code.
+- use static code analysis tools to verify code for issues and potential bugs.
```

Comment:
On diagram mark public fields and methods with + and private ones with -

Reply:
The fields  was marked with corresponding operators.